### ISSUE TITLE
Having issue with custom category

### ISSUE THREAD

**Issue by Noghri1666** [geoffhumphrey/brewcompetitiononlineentry#1080]
Version:

Installation URL: oregonstatefair.brewcomp.com

Is your installation hosted on brewcompetition.com or brewcomp.com?
On  brewcomp.com

Description of Issue:
I've added a custom style, but can't figure out how to force people to list certain special ingredients for that style. Is this even possible? 

For enhancements, please prepend "Enhancement - " to the title. For issues/bugs, please prepend the BCOE&M version number the title.

**Comment by geoffhumphrey** [comment]
Hello @Noghri1666 - to make sure that a style requires special ingredients and/or further information, the _Required Info_ toggle for the style should be checked "yes" and the entry info provided. Go to Admin > Competition Preparation > Styles Accepted >Manage and edit the style.

![image](https://user-images.githubusercontent.com/5216502/61465713-d9908a00-a935-11e9-9681-82e85999cc8f.png)

Since your installation is hosted, please let me know if that doesn't work and I'll investigate further.

**Comment by matdias** [comment]
Same thing happened to me. For some reason the required info form is hidden.

**Comment by geoffhumphrey** [comment]
This was an issue in past versions. Make sure you've updated to the latest version and try again.

**Comment by Beer-Ranger** [comment]
I was the one who brought this up last year under issue #980 and I have upgraded to 2.1.17......and I still have the issue.  The custom styles do have the spade showing extra info is required but the box is not appearing.  I tried the fix in #980 and like last time, it gave me the box, but nothing in that box would be saved.

I don't think I will roll back to 2.1.13 like I did last year, this time we'll just get the brewers to email the extra info and manually add it to the Admin Notes field.
